core/templates/dev/head/pages/library/LibraryFooter.js

```diff
@@ -17,8 +17,9 @@
  */
 
 oppia.controller('LibraryFooter', [
-  '$scope', 'LIBRARY_PAGE_MODES', function($scope, LIBRARY_PAGE_MODES) {
-    $scope.pageMode = GLOBALS.PAGE_MODE;
-    $scope.LIBRARY_PAGE_MODES = LIBRARY_PAGE_MODES;
+  '$scope', '$window', 'LIBRARY_PAGE_MODES', 'LIBRARY_PATHS_AND_MODES',
+  function($scope, $window, LIBRARY_PAGE_MODES, LIBRARY_PATHS_AND_MODES) {
+    var pageMode = LIBRARY_PATHS_AND_MODES[$window.location.pathname];
+    $scope.canDisplayFooter = (pageMode !== LIBRARY_PAGE_MODES.SEARCH);
```

Comment:
`can` is also a verb, maybe `footerCanBeDisplayed`?

[Variable names starting with a verb typically denote function names (as it is supposed to do something).]

Reply:
Done.